### ISSUE TITLE
jsconfig in VSCode as described in docs doesn't work

### ISSUE THREAD

**Issue by mrmowji** [cypress-io/cypress-documentation#3063]
### Current behavior:

I think your docs need some modifications. Currently, in [IDE Integration](https://docs.cypress.io/guides/tooling/IDE-integration.html#Reference-type-declarations-via-jsconfig), setting up a `jsconfig.json` file is mentioned as an alternative way to make Intellisense work in VSCode (instead of adding triple slash directives). But it won't work. In my case I needed a `tsconfig.json` with a `"types": ["cypress"]` prop under `"compilerOptions"`.

### Desired behavior:

I think that "(like VS Code)" in "Reference type declarations via jsconfig" should be removed, since it doesn't work in that editor. And may cause confusion. Instead it'd better to put it under "Reference type declarations via tsconfig".

### Versions

Cypress 4.8.0
VSCode 1.47.3
Ubuntu 18.04.4 LTS

**Comment by jennifer-shehane** [comment]
Are you using JavaScript or TypeScript files for your spec files?

**Comment by mrmowji** [comment]
@jennifer-shehane Thanks for the note. I use JavaScript files.

**Comment by mrmowji** [comment]
The problem was the relative path used for `node_modules` in my project. Still wondering how that relative path worked with `tsconfig.json` and not `jsconfig.json`.

**Comment by luanorlandi** [comment]
Thank you for sharing your resolution @mrmowji

Same thing here, the relative path in `includes` with `./node_modules/cypress` and `compilerOptions` with alias path didn't work together, but when isolated, they work fine. Changing to `node_modules/cypress`, they work together. I also noticed that if adding `"src/**/*"` in `includes`, they all work together, even if it's using relative path `./node_modules/cypress`.

Here is my configuration in case anyone ends up with this problem:

Initial `jsconfig.json` (from Cypress docs):

```json
{
  "include": [
    "./node_modules/cypress",
    "cypress/**/*.js"
  ]
}

```

Then changed to this:

```json
{
  "include": [
    "node_modules/cypress",
    "cypress/**/*.js",
    "src/**/*"
  ],
  "compilerOptions": {
    "baseUrl": ".",
    "paths": {
      "@/*": [
        "src/*"
      ]
    }
  }
}
```

**Comment by hcabnettek** [comment]
@luanorlandi Awesome. This works perfect. I didn't want to use the triple slash method in every spec. Well done sir!

**Comment by hcabnettek** [comment]
> Thank you for sharing your resolution @mrmowji
> 
> Same thing here, the relative path in `includes` with `./node_modules/cypress` and `compilerOptions` with alias path didn't work together, but when isolated, they work fine. Changing to `node_modules/cypress`, they work together. I also noticed that if adding `"src/**/*"` in `includes`, they all work together, even if it's using relative path `./node_modules/cypress`.
> 
> Here is my configuration in case anyone ends up with this problem:
> 
> Initial `jsconfig.json` (from Cypress docs):
> 
> ```json
> {
>   "include": [
>     "./node_modules/cypress",
>     "cypress/**/*.js"
>   ]
> }
> ```
> 
> Then changed to this:
> 
> ```json
> {
>   "include": [
>     "node_modules/cypress",
>     "cypress/**/*.js",
>     "src/**/*"
>   ],
>   "compilerOptions": {
>     "baseUrl": ".",
>     "paths": {
>       "@/*": [
>         "src/*"
>       ]
>     }
>   }
> }
> ```
@luanorlandi Awesome. This works perfect. I didn't want to use the triple slash method in every spec. Well done sir!

**Comment by TheSRK** [comment]
No solutions working for me. jsconfig.json file still showing error

**Comment by jeffnyman** [comment]
I have to agree with the above poster. I'm in the same boat with this in terms of `jsconfig.json` showing an error in VS Code. I used the solution as provided above but I get the error: "JSON schema for a JavaScript project using TypeScript tooling."

I have done nothing with the project except set it up via the following commands:

* npm init -y
* npm install cypress
* npx cypress open

That's it. From there I added the `jsconfig.json` file and put the above suggested contents in place. Everything (to my knowledge) is JavaScript; I'm not sure why "TypeScript tooling" is being indicated.

So far I'm getting a  lot of field reports from classrooms that they are experiencing the exact same thing.
